iOS Example/iOS Example/ViewController.swift

```diff
@@ -56,8 +56,38 @@ class ViewController: UIViewController {
 		tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorButtonText) { self.bounceButton.setTitleColor($0, for: .normal) })
 		tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorBackground) { self.view.backgroundColor = $0 })
 		tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorText1) { self.titleLabel.textColor = $0 })
-		tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorText2) { self.bodyLabel.textColor = $0 })
-
+        tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorText2) { self.bodyLabel.textColor = $0 })
+        
+        // If you used `TweaksCallbacks` type, you can subscribe to those tweaks with callbacks.
+        // Those will be run in order.
```

Comment:
ahhhhh this is a neat feature!

Though: if callbacks are created in completely different places (e.g. maybe one is in some networking code, and another is in a user-manager, and a third is in a view controller) then it's not clear which one is getting created first.

Is there a use case where it matters which closure is getting called first? I think it might be better to allow tweakers to combine multiple closures into a single one, or call methods in-order within a closure — but not allow tweakers to expect that closures are created in a certain order.

(Just pondering here because we're debating whether it'd be OK to use `TweakBinding` instead of `CallbackIdentifier`)